packages/core/src/config/knapsack-pro-env.config.ts

```diff
@@ -211,20 +210,19 @@ export class KnapsackProEnvConfig {
     return process.env.KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE;
   }
 
-  public static get userSeatHash(): string | void {
+  public static get maskedUserSeat(): string | void {
+    const regexp = /(?<=\w{2})[a-zA-Z]/g
+    const maskingChar = "*"
```

Comment:
```suggestion
    const maskingChar = '*';
```

Reply:
@shadre it's less annoying if you [setup the editor](https://github.com/KnapsackPro/knapsack-pro-js#ide-vscode) instead of manually running prettier to fix things ;)